### ISSUE TITLE
Players' job levels often disappear, and it tends to happen to the same players repeatedly.

### ISSUE THREAD

**Issue by KNzE07** [AdvancedPlugins/Jobs#69]
### Describe the bug

As per the topic, I have no idea how this is happening. Players often open tickets to report that their levels have been reset, and their leaderboard scores have also been reset to 0.

My setup consists of 4 servers, all using MySQL as the database. Other plugins are working fine without any issues.

### How to reproduce

I don’t know what’s causing the issue, but please help investigate why players’ levels and leaderboard scores are disappearing. I’ve already set the reset configuration to false, so it shouldn’t be happening.

**Comment by KPGTB** [comment]
Hey! That's really strange, and unfortunately I am not able to reproducte this issue. Is there anything in common between these players who report this issue? Do you use any plugin that modifies UUID? When does this issue happen? I.e. after the server restart, or maybe just after re-joining the server? Also, are there any errors in the console when this happens? Can you send me example logs when this happens? Also, did you enable `bungee-fix` in config?

**Comment by KNzE07** [comment]
I don’t have a system that changes players' UUIDs.
The players' job levels suddenly disappear without them doing anything. When they check again, they find that their job levels are gone.

The issue with job levels seems to occur with players who take on multiple jobs. I suspect that your MySQL database doesn’t store data in separate tables for each specific category, such as a UUID table or a player level table. Instead, it seems like you’re storing it in a JSON-like format, which I think might be causing the problem.

That is just a guess, but normally, data in an SQL database should be stored in tables, not in a JSON format.

**Comment by KPGTB** [comment]
Yeah, I know that our database format is bad, but the entire storage backend was created a few years ago (this plugin was created based on the BattlePass code) by previous developers. Currently, it'd be really hard and problematic to change the current format to a proper one. Also, I don't think that it causes your problem

**Comment by KPGTB** [comment]
Hey! Please check if after the latest updates, there are any warning in the console about these players (i.e. when joining the server by these players or during jobs refresh)
